apps/cowswap-frontend/src/lib/hooks/useApproval.ts

```diff
@@ -11,7 +11,12 @@ import { Nullish } from 'types'
 
 import { useTokenAllowance } from 'legacy/hooks/useTokenAllowance'
 
-import { ApprovalState } from 'common/hooks/useApproveState'
+export enum ApprovalState {
```

Comment:
Don't we have this somewhere else already?
I saw it in a previous PR in this series.

Reply:
Fixed it in the last PR